src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {queryParams.hasOwnProperty('signup') ?
             <p>
-              <Link
-                to={`/${orgId}/${projectId}/?onboarding=1#welcome`}
+              <a
```

Comment:
Do what @benvinegar said. Add a `NOTE` or `TODO` comment here to explain so someone else doesn't change it.